src/platform/bouffalolab/BL702/BUILD.gn

```diff
@@ -41,30 +43,46 @@ static_library("BL702") {
     "../common/KeyValueStoreManagerImpl.cpp",
     "../common/Logging.cpp",
     "../common/PlatformManagerImpl.cpp",
-
-    # "${chip_root}/src/credentials/CHIPCert.h",
-    # "${chip_root}/src/credentials/DeviceAttestationCredsProvider.h",
-
-    # "../common/FactoryDataProvider.cpp",
-    # "../common/FactoryDataProvider.h",
   ]
 
   if (chip_enable_ota_requestor) {
-    sources += [
+    common_sources += [
       "../common/OTAImageProcessorImpl.cpp",
       "../common/OTAImageProcessorImpl.h",
     ]
   }
 
   if (chip_enable_ble) {
-    sources += [
+    common_sources += [
       "../common/BLEManagerImpl.cpp",
       "../common/BLEManagerImpl.h",
     ]
   }
+
+  if (chip_enable_factory_data || chip_enable_factory_data_test) {
+    if (chip_enable_factory_data) {
+      defines = [ "CONFIG_BOUFFALOLAB_FACTORY_DATA_ENABLE" ]
```

Comment:
same comments as for the other file: use config header, use sources directly instead of common_sources.

Reply:
See other comments.